students/brandon_nguyen/session06/test_mailroom4.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Week6 Excercise mailroom part 4b- the test file
+# Student: Brandon Nguyen - Au2018
+import sys
+import time
+import unittest
+import os
+from decimal import Decimal
+from datetime import datetime, date
+from mailroom4 import sort_sum4_report
+from mailroom4 import update_donation
+from mailroom4 import create_letters_text
+from mailroom4 import create_files
```

Comment:
Minor: Better combine lines 10-13 to `from mailroom4 import sort_sum4_report, update_dontaion, create_letters_text, create_files`. It's also OK to do `from mailroom4 import *` in a test suite that's dedicated to the `mailroom4.py` module.